src/Conversion/AutoConversion/AutoConvertStream.tsx

```diff
@@ -47,6 +51,11 @@ class AutoConvertStream extends LocalStorageComponent<Props, State, SavedState>
             <Button onClick={(event: React.MouseEvent<Button>) => this.onClearClick(event)}>Clear</Button>
           </ButtonGroup>
         </ButtonToolbar>
+        <input 
+          type="checkbox"
```

Comment:
Have you tried a `Button` and binding to the `active` state?

https://react-bootstrap.github.io/components/buttons/#buttons-active

Reply:
Then you can put it in the `ButtonToolbar`

Reply:
Actually, it looks like you want:  https://react-bootstrap.github.io/components/button-group/